<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>659</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LOONGSON Auto-test Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/image/logo.png</normaloff>
    <normalon>:/image/logo.png</normalon>
    <disabledoff>:/image/logo.png</disabledoff>
    <disabledon>:/image/logo.png</disabledon>
    <activeoff>:/image/logo.png</activeoff>
    <activeon>:/image/logo.png</activeon>
    <selectedoff>:/image/logo.png</selectedoff>
    <selectedon>:/image/logo.png</selectedon>:/image/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>160</width>
    <height>90</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QTreeView" name="treeView">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="configFilePathBrowser">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="typeLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>测试信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="configTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="helpAction">
   <property name="text">
    <string>关于LOONGAT(&amp;A)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>查看帮助(&amp;A)</string>
   </property>
  </action>
  <action name="action_LOONGAT_A_2">
   <property name="text">
    <string>关于LOONGAT(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
